external/loaders/loaders/mappers/_fine_resolution_budget.py

```diff
@@ -194,13 +216,11 @@ class FineResolutionSources(GeoMapper):
     def __init__(
         self,
         fine_resolution_time_mapping: Mapping[Time, xr.Dataset],
-        offset_seconds: Union[int, float] = 0,
```

Comment:
I'm guessing you removed this arg because it's not used? You sure the change has no effects?

Reply:
In this case, if you look in the code below, you can see `self._offset_seconds` is set and then never used in the code. If someone is using it, they have a bug.